### PR TITLE
Added Windows WSL to installation

### PR DESCRIPTION
**Pull request by Oshanath**

### CODE REVIEW

File: doc/src/installation.md [hunk 1]
```diff
@@ -149,6 +149,81 @@ Note: the miniforge shell seems to be running some version of `git-bash`
...
+* Now clone the LFortran git repository
+```bash
+git clone https://github.com/lfortran/lfortran.git
+cd lfortran
+```
+
+* Run the following commands
+```bash
+conda activate lf
+dos2unix ./build0.sh
```
Comment (wolfv) [inline doc/src/installation.md:200]: I don't think this is necessary if we configure `git` correctly.

E.g. add a `.gitattributes` file in teh root, and put 

```
build0.sh text eol=lf
...
```

https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings#example

Comment (Oshanath) [reply doc/src/installation.md:200]: If we configure git to `eol=lf` wouldn't it be a problem when compiling on Windows natively? Would it be possible to activate the `.gitattributes` file only if we are compiling in Windows via WSL?

Comment (wolfv) [reply doc/src/installation.md:200]: When natively compiling on Windows you'd use the `build0.bat`, right? I think git bash can handle both if you want to use `build0.sh` in git bash.

Comment (Oshanath) [reply doc/src/installation.md:200]: You're right! I will push them now. Thanks.

File: doc/src/installation.md [hunk 2]
```diff
@@ -149,6 +149,75 @@ Note: the miniforge shell seems to be running some version of `git-bash`
...
 
+## Build from Git on Windows with WSL
+* In windows search "turn windows features on or off".
+* Tick Windows subsystem for Linux.
+* Press OK and restart computer.
+* Go to Microsoft store and download Ubuntu 20.04, and launch it.
+* Run the following commands.
+
+```bash
+wget --no-check-certificate https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
```
Comment (wolfv) [inline doc/src/installation.md:160]: https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh

Is the URL I get from https://docs.conda.io/en/latest/miniconda.html

I would recommend miniforge though (https://github.com/conda-forge/miniforge)

Comment (wolfv) [reply doc/src/installation.md:160]: I don't think you need to use the --no-check-certificate. At least you shouldn't :)

Comment (certik) [reply doc/src/installation.md:160]: Yes, miniforge.

**Review by certik — APPROVED** [review]

**Comment by certik** [comment]
The CI failure is unrelated (https://gitlab.com/lfortran/lfortran/-/issues/758).

**Pull request merged**